api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java

```diff
@@ -139,7 +139,7 @@ default List<DbCardCount> findSurveyCounts(SearchTerm searchTerm) {
               + "from DbCriteria c "
               + "where c.standard = :standard "
               + "and c.conceptId in (:conceptIds) "
-              + "and match(c.fullText, concat('+[', :domainId, '_rank1]')) > 0")
+              + "and cast(match(c.fullText, concat('+[', :domainId, '_rank1]')) as integer) > 0")
```

Comment:
did the cast calls work eventually? I thought there were mysql errors

Reply:
it works. The issue is something else